diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8aee89d1..3c71524d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -108,14 +108,7 @@ if (COMPILE_WITH_PYTHON)
     python/sparta_support/Completer.cpp)
 endif ()
 
-execute_process (COMMAND bash "-c" "git describe --tags --always"
-  WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-  OUTPUT_VARIABLE GIT_REPO_VERSION RESULT_VARIABLE rc
-)
-
-if (NOT rc EQUAL "0")
-  message (FATAL_ERROR "could not run git command 'git describe --tags --always', rc=${rc}")
-endif ()
+SET(GIT_REPO_VERSION $ENV{GIT_TAG})
 
 #
 # Debug and Release are cmake internal build types.  Release will add
